requirements.txt

```diff
@@ -1,12 +1,12 @@
 kombu==3.0.30
 Jinja2==2.8
-mozdownload==1.20.2
+mozdownload==1.21
 mozprocess==0.22
 python-jenkins==0.4.8
 redo==1.5
 requests==2.9.1
 taskcluster==0.3.4
-treeherder-client==2.1.0
+treeherder-client==3.0.0
```

Comment:
I cannot use version 3.1.0 because mozdownload has a strict dependency for 3.0.0.